.github/workflows/react-ci.yml

```diff
@@ -0,0 +1,14 @@
+name: React CI
+
+on: pull_request
+  
+jobs:
+  react_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```

Comment:
Can we add `name` field too for these two `uses` , What do you think? It will just make it more clear the steps, like the below :)

`name: Checkout repository`
`name: Setup Node.js`

Reply:
Added :)